WordPress/gutenberg
# Add contextual commands

Related #50407 and #48457

## What?

This PR adds support for contextual commands to the command center by providing:

 - A `setContext` action in the commands store to set the current context.
 - Allowing command loaders and commands to provide a "context" key when registering them. 

The behavior is the following:

 - When you open the command center, the commands of the current context are rendered.
 - When you search the command center, the behavior remains unchanged and any command can be rendered.

For the current PR, I only added a single contextual command: "Browse styles" to the site editor.

## Testing Instructions

1- Open the site editor
2- In view mode, notice that the "browse styles" command is visible as soon as you open the command center
3- In edit mode, notice that the "browse styles" is not visible but is accessible if you search for it.
